styles/global.css

```diff
@@ -11,43 +10,87 @@ body {
 }
 
 body {
-  font-family: Arial, Helvetica, sans-serif;
   line-height: 1.6;
+  font: Arial;
 }
+
 .headerNav {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  position: absolute;
   width: 100%;
-  /* make sure header always overlay bg image */
-  z-index: 2;
+  background-color: black;
+  position: fixed;
+  height: 60px;
+}
+.logo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+.logo,
+.navbar,
+.dropbtn,
+.search-box::placeholder {
+  font-family: "Hind", sans-serif;
+  font-weight: 600;
 }
 
 .logo a {
   text-decoration: none;
-  color: black;
+  color: white;
 }
-.logo h1 {
-  font-size: 24px;
+.logo-h1 {
+  font-size: 16px;
   margin-bottom: 0;
   margin-left: 20px;
+  justify-items: center;
 }
-.logo p {
+.logo-p {
   margin-top: 0;
-  font-size: 14px;
+  font-size: 10px;
+  color: rgb(233, 231, 231);
   text-align: center;
-  margin-left: 20px;
+  margin-left: 16px;
 }
 .navbar {
   display: flex;
   justify-content: space-between;
+  align-items: center;
+  justify-items: center;
   padding-right: 40px;
-  gap: 30px;
+  height: 100%;
 }
+
 .navbar a {
   text-decoration: none;
-  color: black;
+  color: white;
+}
+
+.mystory-header,
+.blogs-header,
+.dropdown {
+  display: flex;
+  border-left: 2px solid white;
+  justify-content: center;
+  align-items: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: 100%;
+}
+
+.dropdown {
+  border-right: 2px solid white;
```

Comment:
is there any specific reason for border-right on dropdown? Would it be better on `search-container`?